### PR TITLE
feat: help centre

### PR DESCRIPTION
**Pull request by zbettenbuk**
image

### CODE REVIEW

File: react/features/help/actions.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Action to set the visibility of the Help View.
+ *
+ * @param {boolean} visible - True if visible, false otherwise.
+ * @returns {{
+ *     type: SET_HELP_VIEW_VISIBLE,
+ *     visible: boolean
+ * }}
+ */
+export function setHelpViewVisible(visible: boolean) {
```
Comment (saghul) [inline react/features/help/actions.js]: This seems to implement the same thing dialogs do. Can't we make openDialog work here? I believe we do the same with the BottomSheet, which is also a sliding view.

File: react/features/help/components/HelpView.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                <View
+                    style = { [
+                        _headerStyles.page,
+                        _styles.page
+                    ] }>
+                    <HeaderWithNavigation
+                        headerLabelKey = 'helpView.header'
+                        onPressBack = { this._onRequestClose } />
+                    <SafeAreaView style = { styles.safeArea }>
+                        <WebView source = {{ uri: 'https://web-cdn.jitsi.net/faq/meet-faq.html' }} />
```
Comment (saghul) [inline react/features/help/components/HelpView.js]: Let's make this configurable please. Configurable as in config.js, that is, with that as a default.

File: react/features/help/components/styles.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+import { ColorSchemeRegistry, schemeColor } from '../../base/color-scheme';
+
+export default {
+    safeArea: {
+        flex: 1
+    }
+};
+
+ColorSchemeRegistry.register('Help', {
```
Comment (saghul) [inline react/features/help/components/styles.js]: Wouldn't this be unneeded if we used a Dialog for this feature?

File: react/features/welcome/components/WelcomePageSideBar.native.js [hunk 4]
```diff
@@ -25,11 +26,6 @@ import styles, { SIDEBAR_AVATAR_SIZE } from './styles';
  */
 const PRIVACY_URL = 'https://jitsi.org/meet/privacy';
 
-/**
- * The URL at which the user may send feedback.
- */
-const SEND_FEEDBACK_URL = 'mailto:support@jitsi.org';
```
Comment (saghul) [inline react/features/welcome/components/WelcomePageSideBar.native.js]: We should still have a button for this outside of the webview, I think. Maybe at the bottom?

Comment (zbettenbuk) [reply react/features/welcome/components/WelcomePageSideBar.native.js]: We agreed to make it part of the faq, I think that's better, especially now that we made the faq url configurable (e.g. for branding)

File: lang/main.json [hunk 5]
```diff
@@ -773,5 +774,8 @@
     "lonelyMeetingExperience": {
         "button": "Invite others",
         "youAreAlone": "You are the only one in the meeting"
+    },
+    "helpView": {
+        "header": "Help centre"
```
Comment (saghul) [inline lang/main.json:779]: While I love our use of en-GB here, the rest of the app is written in "English Simplified" so -> "center" ?

Please note thatt I take no pleasure in writing this :-P

File: react/features/chat/components/native/ChatInputBar.js [hunk 6]
```diff
@@ -136,7 +135,7 @@ class ChatInputBar extends Component<Props, State> {
      */
     _onFocused(focused) {
         return () => {
-            Platform.OS === 'android' && this.setState({
```
Comment (saghul) [inline react/features/chat/components/native/ChatInputBar.js:139]: why the change?
